drivers/exec/driver_test.go

```diff
@@ -160,7 +161,71 @@ func TestExecDriver_StartWaitStop(t *testing.T) {
 		status, err := harness.InspectTask(task.ID)
 		require.NoError(err)
 		require.Equal(drivers.TaskStateExited, status.State)
-	case <-time.After(1 * time.Second):
+	case <-time.After(10 * time.Second):
+		require.Fail("timeout waiting for task to shutdown")
+	}
+
+	require.NoError(harness.DestroyTask(task.ID, true))
+}
+
+func TestExecDriver_StartWaitStopKill(t *testing.T) {
```

Comment:
Confirmed that this test fails in [Job #13860.1](https://travis-ci.org/hashicorp/nomad/jobs/463110275) with a timeout as expected.

```
    testlog.go:32: 2018-12-04T02:07:49.739Z [INFO ] exec/driver.go:284: exec: starting task: driver_cfg="{Command:/bin/bash Args:[-c echo hi; sleep 600]}"
    driver_test.go:229: 
        	Error Trace:	driver_test.go:229
        	Error:      	timeout waiting for task to shutdown
        	Test:       	TestExecDriver_StartWaitStopKill
```